rules/aip0160/filter_field.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0160
+
+import (
+	"fmt"
+
+	"github.com/googleapis/api-linter/lint"
+	"github.com/googleapis/api-linter/locations"
+	"github.com/googleapis/api-linter/rules/internal/utils"
+	"github.com/jhump/protoreflect/desc"
+)
+
+const (
+	messageFmt = `Guidance: use the field, "string filter", not "%s %s"`
+	suggestion = "string filter"
+)
+
+var filterField = &lint.MethodRule{
+	Name: lint.NewRuleName(160, "filter-field"),
+	OnlyIf: func(m *desc.MethodDescriptor) bool {
+		return utils.IsListMethod(m) || utils.IsCustomMethod(m)
+	},
+	LintMethod: func(m *desc.MethodDescriptor) []lint.Problem {
+		var problems []lint.Problem
+		for _, f := range m.GetInputType().GetFields() {
+			if f.GetName() == "filters" ||
+				(f.GetName() == "filter" && utils.GetTypeName(f) != "string") {
```

Comment:
Would you be open to separating these into two separate rules? `synonyms` and `filter-field-type`?

We want these to be more granularly configurable, so that disabling one Linter rule does not disable two "checks" simultaneously.

Reply:
Done, this pr only validates the name and https://github.com/googleapis/api-linter/pull/1524 only validates the type.